src/pages/about/index.js

```diff
@@ -22,20 +121,102 @@ const About = () => {
+                                )}
+                            </Text>
+
+                            <Text secondary color="white">
+                                {localize(
+                                    'Our evolution is powered by over 20 years of customer focus and innovation.',
+                                )}
+                            </Text>
+                        </ContentWrapper>
+                    )}
+                    {active_tab === 'leadership' && (
+                        <ContentWrapper>
+                            <LeadershipWrapper mt="4rem" ai="center">
+                                <Wrapper max_width="28.2rem" margin={{ right: '2.4rem' }}>
+                                    <Image
+                                        width="28.2rem"
+                                        img_name="jean-yves.png"
+                                        alt={localize('Jean Yves')}
+                                    />
+                                </Wrapper>
+
+                                <div>
+                                    <LeadershipTitle
+                                        as="h3"
+                                        size="var(--text-size-header-1)"
+                                        color="white"
+                                    >
+                                        <Localize translate_text="Jean-Yves Sireau" />
+                                    </LeadershipTitle>
+                                    <LeadershipPosition
+                                        as="h4"
+                                        weight="normal"
+                                        color="white"
+                                        lh="3.6rem"
+                                    >
+                                        {localize('Founder and Chief Executive Officer')}
+                                    </LeadershipPosition>
+                                    <Text color="white">
+                                        {localize(
+                                            'Jean-Yves has been an entrepreneur since the age of 20. From 1997 to 1999, he developed the algorithms that would become one of the world’s first trading platforms. He was granted a patent for his binary options trading system in 2007, and granted two more patents in 2011 for systems and methods that enable financial market speculation.',
+                                        )}
+                                    </Text>
+                                </div>
+                            </LeadershipWrapper>
+                        </ContentWrapper>
+                    )}
+                </StyledContainer>
+            </Background>
+            {active_tab === 'story' && <OurStory />}
```

Comment:
isnt this the same as above, we can combine them right?

Reply:
Oh yes I changed it to: 
```
{is_story && <OurStory />}
            {is_leadership && <Leaders />}
```